docs/instances.md

```diff
@@ -4,10 +4,13 @@
 
 This page provides a list of `Websurfx` instances provided by us and our community. 
 
-|URL|Network|Version|Location|Behind Cloudflare?|Maintained By|TLS|IPv6|Comment|
+|URL|Network|Version|Location|Behind Cloudflare?|Status|TLS|IPv6|Comment|
 |-|-|-|-|-|-|-|-|-|
-|https://websurfx.co/|www|edge|🇺🇸 US|||✅|❌||
-|https://websurfx.onrender.com/|www|edge|🇺🇸 US|||✅|❌||
-|https://alamin655-websurfx.hf.space/|www|v0.21.4|🇺🇸 US||[websurfx project](https://github.com/neon-mmd/websurfx)|✅|❌||
+|https://websurfx.co|www|rolling|🇺🇸 US||<a href="https://status.websurfx.pp.ua"><img src="https://status.websurfx.pp.ua/api/badge/2/status"></a>|✅|❌||
+|https://alamin655-spacex.hf.space|www|rolling|🇺🇸 US||<a href="https://status.websurfx.pp.ua"><img src="https://status.websurfx.pp.ua/api/badge/5/status"></a>|✅|❌||
+|https://websurfx.instance.pp.ua|www|rolling|🇺🇸 US||<a href="https://status.websurfx.pp.ua"><img src="https://status.websurfx.pp.ua/api/badge/7/status"></a>|✅|✅||
+|https://alamin655-websurfx.hf.space|www|stable|🇺🇸 US||<a href="https://status.websurfx.pp.ua"><img src="https://status.websurfx.pp.ua/api/badge/6/status"></a>|✅|❌||
```

Comment:
```suggestion
|https://websurfx.co|www|rolling|🇺🇸 US||[100%](https://status.websurfx.pp.ua)|✅|❌||
|https://alamin655-spacex.hf.space|www|rolling|🇺🇸 US||[80%](https://status.websurfx.pp.ua)|✅|❌||
|https://websurfx.instance.pp.ua|www|rolling|🇺🇸 US||[99%](https://status.websurfx.pp.ua)|✅|✅||
|https://alamin655-websurfx.hf.space|www|stable|🇺🇸 US||[100%](https://status.websurfx.pp.ua)|✅|❌||
```

I think for the uptime readings for all the instances, I would suggest having a text link over the batch. When new instances will be added by other folks in the future, they may prefer to use different uptime monitoring applications for their instances. So using this approach will allow them to easily direct the user to their monitoring site instead of using a single monitoring app used by the project, which will in turn make the addition of new instances easier I feel. :slightly_smiling_face:

Reply:
Here, the uptime shows whether the instance is running or not. By using a random percentage, it neglects the meaning of it. Also, we can just add their instance to the website to monitor their uptime.

Reply:
> Here, the status shows whether the instance is running or not. By using a random percentage, it neglects the meaning of it. Also, we can just add their instance to the website to monitor their status.

Sorry for the late reply.
Yes, I agree that adding a random number is not a good idea, but actually adding each instance manually would be a very time-consuming and tedious process. Additionally, it will also make contributing more instances by the community more lenghty and complicated too. So that's why I was suggesting to just provide a way where people could just put a link to their own uptime monitoring dashboards, as this will make contributing more instances easier.

Reply:
It's simple; all you need to do is copy and paste the link. It won't take long at all. Additionally, they can just link it there if they wish to utilize their own monitoring application.

Reply:
> It's simple; all you need to do is copy and paste the link. It won't take long at all. Additionally, they can just link it there if they wish to utilize their own monitoring application.

Ok, I see, then it is fine. So I would suggest leaving this as it is. No need to make the changes then.